### DISCUSSION TITLE
Timer for NVR

### DISCUSSION THREAD

**Discussion by swamp2k** [roflcoopter/viseron#707] [Q&A]
Hey guys, was wondering if there is a timer option, to stop the NVR feature.
Like, our cam is mainly to record and document autistic behaviour... which is only during daytime. 

I can't seem to find anything related to time in Viseron, amd I missing it?
My alternative would be to stop/start the docker running Viseron.

**Comment by roflcoopter** [comment]
Interesting use case!

You can stop/start cameras on demand over MQTT.
In the future I will make sure to add it to the ui and api as well but right now MQTT is your only option. Would that work for you?

**Reply by swamp2k** [reply]
Oh. MQTT, neat. 
I did notice I could stop/start “something” from Viseron in MQTT but I guess I was not sure what it was. 

I’ll give it a shot. It would tie in nicely to home assistant, so I know where all my schedules are. That would be a big plus 👍

I’ll update when I get a chance to test it. Maybe could help others out in the future.

**Reply by roflcoopter** [reply]
Oh you are using Home Assistant! Viseron supports MQTT discovery so all you need to do is toggle the switch entity that is created
